### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by juanpa097**
### Summary
This is the first official stable release of the Treepat parser and implements the following:
- Treepat grammar using `antlr`.
- Node Operator
- Dot Operator
- Sibling Operator
- Child Operator
- Breath Closure
- Integration tests
- Created `README`

### Test Plan
`mvn test`

### Related Issues
N/A

### CODE REVIEW

File: src/main/kotlin/antlr/treepat/Treepat.g4 [hunk 1]
```diff
@@ -1,15 +1,8 @@
-// Define a grammar called Hello
+// Define a grammar called Treepat.
```
Comment (juanpa097) [inline src/main/kotlin/antlr/treepat/Treepat.g4]: @jpavlich This is the updated grammar for Treepat. We are really concerned with the quality as we don't have much experience writing these type of code. Can you give use a review of the file and maybe give use some feedback regarding naming, format, readability, maintainability, etc.

We would also appreciate if you could review the `TreeFormat.g4` file. 😁

Thank you! 😊

**Comment by johan-smc** [comment]
Why we have conflicts with the master?

**Comment by juanpa097** [comment]
> Why we have conflicts with the master?

I have no idea... apparently we did a hotfix and that produced the conflicts.

**Comment by juanpa097** [comment]
This PR is closed do to merge conflicts, this conflicts are going to be solved in the branch `release/1.0`. The new PR is #66

**Pull request closed without merging**